x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -265,6 +266,46 @@ export function registerIndexRoutes({
     })
   );
 
+  router.post(
+    {
+      path: '/internal/enterprise_search/indices/{indexName}/ml_inference/pipeline_processors',
+      validate: {
+        params: schema.object({
+          indexName: schema.string(),
+        }),
+        body: schema.object({
+          pipeline_name: schema.string(),
+          model_id: schema.string(),
+          source_field: schema.string(),
+          destination_field: schema.maybe(schema.nullable(schema.string())),
+        }),
+      },
+    },
+
+    elasticsearchErrorHandler(log, async (context, request, response) => {
+      const { client } = (await context.core).elasticsearch;
+      const {
+        model_id: modelId,
+        pipeline_name: pipelineName,
+        source_field: sourceField,
+        destination_field: destinationField,
```

Comment:
I'll wait for the linter to say something about this; last time I saw it was complaining about the non-alphabetical order of properties.